src/lightning/fabric/accelerators/registry.py

```diff
@@ -73,14 +73,14 @@ def register(
         data["description"] = description
         data["init_params"] = init_params
 
-        def do_register(name: str, accelerator: Callable) -> Callable:
+        def do_register(accelerator: Callable) -> Callable:
```

Comment:
lets pls add a test to reproduce your reported casem and demonstrate that this change fixes it 🦩

Reply:
The tests have been updated. I’d be glad if you could take a look at them